src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Ssl.cs

```diff
@@ -399,7 +399,7 @@ public static bool SslCheckHostnameMatch(SafeSslHandle handle, string hostName,
                 case 1:
                     return true;
                 default:
-                    if (NetEventSource.IsEnabled)
+                    if (NetEventSource.Log.IsEnabled())
```

Comment:
Ideally we'd use `.Log.` with the actual logging methods as well, e.g. Error, Info, etc.  But, as that's not necessary for the trimming, and as I'm hoping all of this NetEventSource stuff either goes away entirely or at least gets merged into the new event sources soon, it's not worth doing in this PR.  So just calling it out, but no action required.